downstream/assemblies/platform/assembly-aap-upgrade-to-latest.adoc

```diff
@@ -0,0 +1,20 @@
+
+ifdef::context[:parent-context: {context}]
+
+[id="aap-upgrade-to-latest"]
+= {PlatformName} {PlatformVers} Upgrades
```

Comment:
```suggestion
= {PlatformName} {PlatformVers} upgrades
```

Reply:
Title doesn't match the ID or filename. I would just change the ID and filename to "aap-upgrades".